### DISCUSSION TITLE
770

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#940] [Comments]
# 770

사내에 공유드렸던 내용을 기록차 남겨둡니다. 저는 혼란한 상황이나 환경을 계속해서 체계적으로 정리/정돈하는 것은 2순위이고, 1순위는 혼란한 상황/환경 내에서 적응하는 것이 가장 중요하다고 생각하는데요. 혼란함 그 자체를 못견디는 것은 계속해서 변하는 업계에서는 꼭 해결해야할 역량이라고 봐요. 혼란함이 있을때마다 매번 정리 정돈하고, 체계적으로 모든 것을 하려고하면 오히려 변화의 속도를 못쫓아가고 항상 뒤쳐지게 된다고 생각합니다. 빚이 없는 상태가 가장 마음이 편하겠지만, 목표를 달성하기 위해 빚을 내 대출자로서의 삶을 견뎌야하는 것처럼요. 절대 빚을 지지 않고 원하는 목표를 달성하려면 수십년이 걸릴 수 있겠죠. 개인이 감당 가능한 범위 내에서 빚을 내고 목표에 빠르게 도달하는 것을 레버리지 라고하는데,..

https://jojoldu.tistory.com/770

**Comment by hhamja** [comment]
좋은 내용 감사합니다. 
저는 이상하게 안정된 공간에 있으면 지루하고 시간이 아깝더라고요 ㅎㅎ
향로님도 그런가요?
